packages/kmath/src/angles.ts

```diff
@@ -1,22 +1,21 @@
-import {clockwise} from "../../../util/geometry";
+// This file contains helper functions for working with angles.
 
-import type {Coord} from "@khanacademy/perseus";
-import type {vec} from "mafs";
```

Comment:
`angles` needs to be moved out of `perseus` for scoring reasons. `maf` can't be in part of the process because it depends on React.